plugins/explore-backend/README.md

```diff
@@ -76,19 +76,18 @@ To index explore tools you will need to register the search collator in the
 ```diff
 +import { ToolDocumentCollatorFactory } from '@backstage/plugin-explore-backend';
 
-async function createSearchEngine(
```

Comment:
It doesn't look this this has changed in the example app, so it's possible this was my mistake. I may have copied it from elsewhere but don't see the issue anywhere else. This probably should have been `createPlugin`. I think this makes more sense, as maintaining examples in this way that reference parent scopes (e.g. the parent function(s)) are hard to maintain and can go stale easier. So it seems better to avoid this at the hopefully small risk someone might not realize exactly where to change the code.